testsuite/features/step_definitions/docker_steps.rb

```diff
@@ -24,11 +24,11 @@ def retrieve_build_host_id
 
 # OS image build
 When(/^I navigate to images webpage$/) do
-  visit("https://#{$server.full_hostname}/rhn/manager/cm/images")
+  step %(I follow the left menu "Images > Image List")
```

Comment:
I'd directly use the step inside. As we don't process any data on this step definition.

Reply:
Good point.